generators/server/templates/app/src/main/resources/application.yml

```
@@ -0,0 +1,66 @@
+################ Actuator #####################
+management:
+  endpoints:
+    web:
+      exposure:
+        include: configprops,env,health,info,logfile,loggers,metrics,prometheus
+  endpoint:
+    health:
+      show-details: always
+
+################### Database ###################
+spring:
```

YAMLlint (1.37.1)

[error] duplication of key "spring" in mapping

(key-duplicates)